.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
```

Comment:
It should be 'Propose a talk/workshop', as we are mentioning about both in about section.